declarations_site/spotter/templatetags/spotter_tags.py

```diff
@@ -9,6 +9,8 @@ def uk_plural(value, arg):
     args = arg.split(',')
     value = int(value)
     rem = value % 10
+    if value > 100:
```

Comment:
It's not an particular app function, so this dependence looks at least strange. Ok where should I leave it?
- in spotter/tags
- in chatbot/utils